input/fsh/Clinical datapoints profiles/BodyMeasures.fsh

```diff
@@ -0,0 +1,19 @@
+Profile: BodyHeight
+Parent: StandardBodyHeight
+Id: ichom-body-height
+Title: "Body height"
+Description: "The height of a person diagnosed with breastcancer"
```

Comment:
We can drop the `The`, just like we don't have it in the questionnaires. In light of this, the link in the patient profile description doesn't look to be too helpful either.

![image](https://user-images.githubusercontent.com/110988/177744157-bf012830-de8a-4591-adb1-48e122729974.png)